tools/purge_omsagent.sh

```diff
@@ -0,0 +1,407 @@
+#! /bin/bash
```

Comment:
1. should check that we are running as root and warn+exit if not
2. what's your plan for shipping this file, will this be bundled a-la the TST? customers will have to DL from GitHub?

Reply:
1. Yes sorry I was going to add that but forgot
2. The second, I was just going to have it as a standalone that customers can download directly from GitHub

Reply:
1. Fixed in this commit: https://github.com/microsoft/OMS-Agent-for-Linux/pull/1255/commits/fb08aac1fb438c916c00116b69ab79411553a011